### ISSUE TITLE
Get the app to F-Droid

### ISSUE THREAD

**Issue by JarlavGrenland** [AnimeTwist/twist-mobile#19]
Add the App to the F-droid app store

Possibly in addition to  #17

**Comment by ATechnoHazard** [comment]
Hi, this is something I considered at the very beginning, but I don't think it's possible considering the app requires secret keys in the build process, which as far as I know, Fdroid does not have support for.

**Comment by JarlavGrenland** [comment]
> 
> 
> Hi, this is something I considered at the very beginning, but I don't think it's possible considering the app requires secret keys in the build process, which as far as I know, Fdroid does not have support for.

I suppose you could setup you own repo https://f-droid.org/en/docs/Setup_an_F-Droid_App_Repo/ or use another 3-party?

**Comment by JohnDowson** [comment]
Yeah there isn't a trusted way to get the app if you don't provide builds on github or put it on fdroid.
